### PR TITLE
Ruby script to list and update files which has force_ssl and apply_me…

### PR DESCRIPTION
**Pull request by poornima-moj**
…thod strings. 

To update RDS module to 5.4

Related to: https://github.com/ministryofjustice/cloud-platform/issues/1563

### CODE REVIEW

File: bin/rds-force_ssl-list-update.rb [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    stdout.split("\n")
+  end
+end
+
+def tfdir(namespace)
+  "#{NAMESPACE_DIR}/#{namespace}/resources"
+end
+
+############################################################
+# a = force_ssl = true, b = force_ssl = "true", c = force_ssl = "false"
```
Comment (digitalronin) [inline bin/rds-force_ssl-list-update.rb:95]: I don't understand what this comment line is trying to tell me

File: bin/rds-force_ssl-list-update.rb [hunk 2]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env ruby
+
+# This script takes a action and option
+# act: list, update
```
[the remark this answers is not in the document]
Comment (poornima-moj) [reply bin/rds-force_ssl-list-update.rb]: Updated the comment

File: bin/rds-force_ssl-list-update.rb (hunk 1 above)
Comment (poornima-moj) [reply bin/rds-force_ssl-list-update.rb]: Its a copy paste comment. Not really needed so removed

**Comment by poornima-moj** [comment]
> This looks good, Poornima. I've made a couple of small suggestions below.
> 
> Also, you use both "action" and "act" to refer to the same thing, in the code and in the comments. I'd recommend picking one and using it consistently - it will make the script easier to understand.

@digitalronin , thanks for your comments. I have used "action" consistently and update the code.

**Pull request merged**